### PR TITLE
complete kata 8 patch article by id

### PR DESCRIPTION
**Pull request by kmgrey**
Complete patch article votes by id
-add test for positive votes and for negative votes
-check for bad vote input and bad article id

### CODE REVIEW

File: controllers/api.controllers.js [hunk 1]
```diff
@@ -54,3 +54,16 @@ exports.postArticleComment = (request, response, next) => {
 		})
 		.catch(next);
 };
+
+exports.patchArticleById = (request, response, next) => {
+	const { article_id } = request.params;
+	const { inc_votes } = request.body;
+	if (typeof inc_votes !== 'number') {
```
Comment (MRombi) [inline controllers/api.controllers.js:61]: Psql will throw an error in this case, no need to check it here. Additionally this type of logic should not be in the controller 🙂

File: models/api.models.js [hunk 2]
```diff
@@ -76,3 +76,18 @@ exports.postCommentByArticleId = (username, body, id) => {
 		});
 	});
 };
+
+exports.updateArticleVotes = (votes, id) => {
+    if (isNaN(id)) {
+		return Promise.reject({ status: 400, msg: 'Bad Request' });
```
Comment (MRombi) [inline models/api.models.js:82]: This would be handled by psql as well 🙂

**Review by MRombi — COMMENTED** [review]
Implement the feedback, well done overall 🙂
